src/vs/workbench/parts/extensions/electron-browser/extensionTipsService.ts

```diff
@@ -61,6 +57,9 @@ export class ExtensionTipsService extends Disposable implements IExtensionTipsSe
 	private _availableRecommendations: { [pattern: string]: string[] } = Object.create(null);
 	private _allWorkspaceRecommendedExtensions: string[] = [];
 	private _dynamicWorkspaceRecommendations: string[] = [];
+	private _allIgnoredRecommendations: string[] = [];
```

Comment:
We can make do with the other 2 arrays for ignore recommendations. Do we really need this as well?